command-snapshot.json

```diff
@@ -1,12 +1,12 @@
 [
-    {
-        "command": "force:limits:api:display",
-        "plugin": "@salesforce/plugin-limits",
-        "flags": [
-            "apiversion",
-            "json",
-            "loglevel",
-            "targetusername"
-        ]
-    }
-]
\ No newline at end of file
+  {
+    "command": "force:limits:api:display",
+    "plugin": "@salesforce/plugin-limits",
+    "flags": ["apiversion", "json", "loglevel", "targetusername"]
+  },
+  {
+    "command": "force:limits:recordcounts:display",
+    "plugin": "@salesforce/plugin-limits",
+    "flags": ["apiversion", "json", "loglevel", "targetusername", "sobjecttypes"]
```

Comment:
Hi @mkreth .  I approved this change prematurely!  Can you change the flag name to "--sobjecttype" (singular)?  I missed that your rename was plural earlier today. Thanks!

Reply:
Ok, I've changed the flag name to singular.